src/features/company/components/Company.tsx

```diff
@@ -3,15 +3,13 @@ import { Peers } from "../../peers";
 import { useSelector } from "react-redux";
 import { Loading } from "../../loading";
 import "./Company.css";
-import { GlobalState } from "../../../store";
+import { AppState } from "../../../store";
 
 export const Company = () => {
   const { selectedCompanyOverview } = useSelector(
-    (state: GlobalState) => state.companyOverviewData
-  );
-  const { selectedTopPeers } = useSelector(
-    (state: GlobalState) => state.peerData
+    (state: AppState) => state.companyOverviewData
   );
+  const { selectedTopPeers } = useSelector((state: AppState) => state.peerData);
```

Comment:
It's possible to export a specially typed `useSelector` from `store` to cut down on the typing here... In store:

```
import { _useSelector, TypedUseSelectorHook } from 'react-redux';
...
export const useSelector: TypedUseSelectorHook<AppState> = _useSelector;
```

and then in here...
```
import { useSelector } from '../../../store';
...
const blah = useSelector(state => state.peerData);
```

Reply:
Great - that looks a lot neater!

When I change `companyOverviewData` useSelector, it can't seem to find the object properties but does for `peerData`. 

Any idea how to solve this? (I'm assuming its something I've changed in `company\redux\actions.ts` which `peers\redux\actions.js` doesn't have)

Reply:
It's getting the type from the return type of the reducer. So if the reducer is in js it's just getting `any`. Could that be it?

Reply:
I'll put this as an `issue` and implement it in another PR :)